ansible_base/authentication/utils/claims.py

```diff
@@ -268,8 +273,96 @@ def update_user_claims(user: Optional[AbstractUser], database_authenticator: Aut
     return user
 
 
+def process_organization_and_team_memberships(results):
+    # Extract organizations where the user is a member
+    org_list = [org_name for org_name, is_member in results['claims']['organization_membership'].items() if is_member]
```

Comment:
@john-westcott-iv is that what we want? and if so, presumably we want to make all teams too?

Reply:
Creating orgs/teams takes time during the login process and we have some customers who have hundreds or thousands of these. So if we limit this to just what the user is a member of it will minimize the impact on any one given user.

Ideally, we would deal with this in a better way. Like maybe there should be a button on an authenticator that says "create assets" so if you have a ton of them an administrator can "click the button" after they configure all of the authenticator maps. Maybe that can be a 2.5-next thing. Even if we did that I think this code will still remain in place because if a user attempts to login in between the admin completing the authenticator maps and clicking the button we would still need to create the org/team for the user.